topics/wordpress/index.md

```diff
@@ -1,14 +1,19 @@
 ---
-aliases: wordpress-development, wordpress-site
+aliases: wordpress-development, wordpress-site, wordpress-theme, wordpress-plugin,automattic,woocommerce
```

Comment:
these seem more "related" than aliases. What do you think about putting them in a related list?
```suggestion
aliases: wordpress-development, wordpress-site
related: wordpress-theme, wordpress-plugin, automattic, woocommerce
```